lib/manageiq/appliance_console/message_configuration_server.rb

```diff
@@ -9,12 +9,16 @@ class MessageServerConfiguration < MessageConfiguration
       attr_reader :jaas_config_path,
                   :server_properties_path, :server_properties_sample_path,
                   :ca_cert_srl_path, :ca_key_path, :cert_file_path, :cert_signed_path,
-                  :keystore_files, :installed_files
+                  :keystore_files, :installed_files, :message_persistent_disk
+
+      PERSISTENT_DIRECTORY = Pathname.new("/var/lib/kafka_persistent_data").freeze
```

Comment:
Should this be under `/var/lib/kafka` or a directory owned by the RPM?

Reply:
Address by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/158/commits/f7a5672bd6a1928f926c060c9a7aea435c0ca26a